hazelcast/datadog_checks/hazelcast/check.py

```diff
@@ -0,0 +1,50 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import AgentCheck
+
+from . import utils
+
+
+class HazelcastCheck(AgentCheck):
+    __NAMESPACE__ = 'hazelcast'
+    SERVICE_CHECK_CONNECT = 'can_connect'
+    SERVICE_CHECK_MC_CLUSTER_STATE = 'mc_cluster_state'
+
+    def __init__(self, name, init_config, instances):
+        super(HazelcastCheck, self).__init__(name, init_config, instances)
+
+        self._mc_health_check_endpoint = self.instance.get('mc_health_check_endpoint', '')
```

Comment:
Should `mc_health_check_endpoint` be in the example conf.yaml? I can't find this option in it.